.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+	"rules": {
+		"indent": ["error", 2],
+		"semi": ["error", "always"],
+		"no-undef": "error",
+		"no-unused-vars": ["error", { "vars": "all", "args": "after-used" }],
+		"eqeqeq": "error",
+		"quotes": ["error", "single"],
+		"no-console": "error",  
+		"comma-dangle": ["error", "never"],
+		"max-len": ["error", { "code": 82, "tabWidth": 4, "ignoreUrls": true }],
```

Comment:
Tabs should be 2 spaces, which is consistent with the indentation setting being used.